docs/reference/cheatsheet.md

```diff
@@ -560,6 +560,21 @@ must also include a table title. See example above.
   - ```{important} This is an example
     of an important directive.
     ```
+* - ````md
+    ```{admonition} Title
+    :class: admonition warning
+    text
+    ```
+  - ````md
+    ```{admonition} This is a title
+    :class: admonition warning
```

Comment:
I don't believe you need to add the `admonition` class - it should automatically be added by the `admonition` directive

Reply:
The admonition directive add a custom class for this particular admonition (and then, you need to add css). Here it is an example of replacing with another already defined class (here the class for warnings) so that you don't need extra css. This is the nice possibility that was not mentioned in the doc: it is actually very easy to write an admonition (with title) with styles of other admonitions without title (warning, tip, caution, etc.)

Reply:
agreed - my point is that you are manually adding both the `warning` class, and an extra `admonition` class, but I don't believe that the extra `admonition` class is needed because it is already added. So I think you can just do:

````
```{admonition} Some title
:class: warning

Some text
```
````

and the `admonition` class will be added to the result anyway

Reply:
Oups sorry, I didn't get your comment! You're right!

Reply:
Replacement done. It's even nicer like that!